packages/fxa-event-broker/lib/notificationProcessor.ts

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { SQS } from 'aws-sdk';
+import { Consumer } from 'sqs-consumer';
+import * as Joi from 'typesafe-joi';
+
+import { Logger } from 'mozlog';
```

Comment:
nit: `mozlog` is a module liek the rest above, should the new line be after it?

Reply:
Yup.